packages/terra-tabs/tests/nightwatch/TabPane/tab-pane-spec.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+
+module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
```

Comment:
can we write these in wdio. There are several examples in terra-core now (badge is in master). I can provide assistance if needed.

Reply:
I've got the tab-pane tests switched over. Working on the tabs tests.